pkg/machines/actions/store-actions.es6

```diff
@@ -44,8 +44,29 @@ import {
  *
  *  The naming convention for action creator names is: <verb><Noun>
  *  with the present tense.
+ *
+ *  New action creators should follow the FSA pattern:
+ *  See: https://github.com/acdlite/flux-standard-action
+ *  They can be created with the makeActionCreatorFSA helper.
  */
 
+function makeActionCreator(type, ...argNames) {
+    return function (...args) {
+        const action = { 'type': type }
+        argNames.forEach((arg, index) => {
+            action[argNames[index]] = args[index]
+        })
+        return action
+    }
+}
+
+function makeActionCreatorFSA(type, ...argNames) {
+    return function (...args) {
+        const action = { 'type': type, 'payload': args[0] }
```

Comment:
similar

Reply:
The returned function expects just a single argument. If this would be the way to go, let's make it clear at the first glance.